### ISSUE TITLE
WebPage not updated auto

### ISSUE THREAD

**Issue by qbihay** [danxfisher/MeetEasier#5]
Hello,

First at all, a big thanks for your works. 

I have a small issue and I don't know why. 
On the page, I can see all the room but it's not refresh automatically.
The config/controller.js.,it is set to 1 minute (the default value). 

Regards

**Comment by jtrimble613** [comment]
Yes, this is an awesome tool, thanks!  I also don't seem to be able to get refresh working.

JT

**Comment by qbihay** [comment]
I write a little workaround to have the auto-refresh work. 
On the html page, I've added the <meta http-equiv="refresh" content="30">

**Comment by DrEverythingllBeAlright** [comment]
in ui-react/package.json file and /ui-react/src/config/flightboard.config.js, change localhost to ip address; i had overlooked this as well

**Comment by 9uan9** [comment]
I updated package.json and flightboard.config.js using ip address and it is still not refreshing.  Is there anything else that I should be looking at that may be causing this to not work for me?

**Comment by danxfisher** [comment]
Try with the most recent release and see if this fixes any issues.  Thanks.

**Comment by jtrimble613** [comment]
Thanks!  The latest version seems to update fine!
